Radius(x).py

```diff
@@ -6,31 +6,68 @@
 @author: amelielaurens
 """
 
+# Predict the radius of the jet in steady state as a function of the axial coordinate x
+
 from models_RJS import *
 import numpy
 import matplotlib.pyplot as plt
 
-# predict the radius of the jet in steady state as a function of the axial coordinate x
-surface_tension =float(input("Enter the surface tension in kg/s^2 : "))
-Rc = float(input("Enter the radius of the collector in m : "))
-s0 = float(input("Enter the radius of the reservoir in m : "))
-discretisation = int(input("Enter an int for the mesh's thinness : "))
+
+discretisation = 20 #The higher the discretisation number is, the finer the discretisation will be, there will be more points on the graphic
+
+
+#Choose one machine and one polymer
+#Machines
+#Super Floss Maxx
+#s0 = 0.06985
+#Rc = 0.3302
+#orifice_radius = 0.001512
+#omega = 57.5
+
+#CANDY-V001
+s0 = 0.0635
+Rc = 0.254
+orifice_radius = 0.000267
+omega = 40.
+
+
+#Polymers
```

Comment:
Same as for the machine, a file with the polymer parameters would be more practical to make change and we could have a database of polymers to choose from.